app/d3plus.js

```diff
@@ -187,6 +268,11 @@ export default {
     }
   },
   legendPosition: "bottom",
+  legendTooltip: {
+    title(d) {
+      return findTooltipTitle(d, this)
+    }
+  },
```

Comment:
There's actually a more succinct way to do this (no pun intended), and it has to do with named functions versus anonymous functions.

As an example, fat arrow functions are _anonymous_, meaning they do not have a name, and thus cannot be referenced anywhere else in code. In this case, `this` is the context of whatever it's parent function is:
```js
title: () => console.log(this); // parent "this"
```

When you "name" a function, is will create it's own local Object defined as `this`:
```js
title: function title() {
  console.log(this); // a new local "this"!
}
```

Adding to the confusion, there is a shorthand in ES6 where if your function has the same name as the key you are assigning it inside of an Object (which is what you are doing, enabling you to grab `this`, which is the internal d3plus config):
```js
title() {
  console.log(this); // a new local "this"!
}
```

But how is `this` the d3plus object? Well, inside of d3plus, when it runs your function, it does this:
```js
title.bind(d3plusObject)(d);
```

Before running the function, we use `bind` to _override_ the `this` variable!

So now, with that in mind, I believe you can change these 2 title configs to be this:
```js
title: findTooltipTitle
```

And then, change your functions to use `this` instead of passing around the object as an argument:
```js
export const findTooltipTitle = (d) => {
  const {item, itemId, parent, parentId} = getTooltipTitle.bind(this)(d);
  const aggregated = Array.isArray(parent) ? "Valores" : parent;
  const title = Array.isArray(item) ? `Otros ${aggregated || "Valores"}` : item;
  const itemBgImg = parentId;
  let bgColor = findColorTooltip(itemBgImg, d);
  let imgUrl = findIcon(itemBgImg, d);
  return tooltipTitle(bgColor, imgUrl, title, parentId);
}
```

Reply:
It didn't work, in `findTooltipTitle` the `this` is `undefined` (I'm doing a `console.log` before the `getTooltipTitle`) and the same thing happens in `getTooltipTitle`

Reply:
OK! Then let's not worry about it 😏 

_"if it ain't broke, don't fix it"_